torch_geometric/sampler/neighbor_sampler.py

```diff
@@ -164,6 +164,8 @@ def __init__(
         else:  # self.data_type == DataType.remote
             feature_store, graph_store = data
 
+            self.edge_time = None  # TODO(kgajdamo): only for the tests to pass
```

Comment:
Can you clarify?

Reply:
This PR was added before I added support for edge-level sampling. I forgot to remove that.